WordPress/Classes/Extensions/Media+Blog.swift

```diff
@@ -0,0 +1,38 @@
+import Foundation
+
+extension Media {
+
+    /// Inserts and returns a new managed Media object, in the context.
+    ///
+    class func inserted(into context: NSManagedObjectContext) -> Media {
```

Comment:
looking more at this and the Swift 3.0 guide, these feel more like a factory methods, so we probably should name then makeMedia(into:), makeMedia(blog:)